README.adoc

```diff
@@ -0,0 +1,250 @@
+* Follow {instagram-url}[@shabad_os on Instagram] and {twitter-url}[@shabad_os on Twitter] and let us know what you think!
+
+## Related Projects
+
+Projects in the Shabad OS ecosystem of free and open source software which interact with or are used by {project} include:
+
+* link:https://github.com/ShabadOS/desktop[Shabad OS for Desktop] allows users to search, present, and live stream the database
+* link:https://github.com/ShabadOS/mobile[Shabad OS for Mobile] is the Android and iOS equivalent of the desktop app
+* link:https://github.com/ShabadOS/api[api] draws data from the database for querying online / for mobile apps
+* link:https://github.com/ShabadOS/gurmukhi-utils[Gurmukhi Utils] is used to swap input/output between ascii/unicode gurmukhi
+* link:https://github.com/GurbaniNow/gurmukhi-fonts[GurbaniNow/gurmukhi-fonts] are used to display bani and gurbani
+
+## Code of Conduct
+
+Please note that this project is released under the Contributor Covenant. By participating in this project you agree to abide by its terms. Our intention is to signal a safe open-source community by welcoming all people to contribute, and pledging in return to value them as whole human beings and to foster an atmosphere of kindness, cooperation, and understanding.
+
+> We as members, contributors, and leaders pledge to make participation in our community a harassment-free experience for everyone, regardless of age, body size, visible or invisible disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
+
+> We pledge to act and interact in ways that contribute to an open, welcoming, diverse, inclusive, and healthy community.
+
+> link:https://www.contributor-covenant.org/version/2/0/code_of_conduct/[The Contributor Covenant]
+
+## License
+
+IMPORTANT: Applies to code and content resting outside of the `data` folder.
+
+The code of the `{repo}` repo is under v3 of the link:LICENSE.adoc[GPL]. It is similar to the Golden Rule: do unto others as you would have them do unto you. In exchange for benefitting from the work completed in this repo, others must share their derivative work under v3 of the link:LICENSE.adoc[GPL].
+
+> This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+
+> This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+> You should have received a copy of the GNU General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+## Gurbani and Panthic Compilations
```

Comment:
Compositions

Reply:
Compilations is okay here

Reply:
We agreed to change sources to Compositions.

The other place where compilation makes sense is where Bani compilation is mentioned, because some banis are made up of different shabads.

But here, it means `Gurbani and Panthic Sources` (under the old terminology), so it should be Compositions.